src/renderer/three/components/column/column.ts

```diff
@@ -0,0 +1,14 @@
+// Author: Fyrestar https://mevedia.com (https://github.com/Fyrestar/THREE.InfiniteGridHelper)
+
+import { BoxGeometry, MeshBasicMaterial, Mesh } from 'three';
+import { ColumnOptions } from './types';
+
+const Column = ({ width, height, coorX, coorY, color }: ColumnOptions) => {
```

Comment:
`coorX` and `coorY` would be the coordinates of the center of the column, or one of the corners?

Reply:
the center of the column